ui/demoBot.html

```diff
@@ -84,8 +84,10 @@
 
 
     var botcoord;
-    postMove('http://localhost:5000/prediction', coord,
-      function(data) {botcoord = JSON.parse(data)
+    var port = window.location.port;
+    var host = window.location.host;
+    postMove("//" + host + (port=="8081" ? "" : ":8081") + "/prediction", coord,
```

Comment:
It will work but, only if the page is being served from the root of the server on the same port as the predictions server. If a separate web server (or a `file://`) is being used, this may not be the case.